src/foundry/common/constants.d.mts

```diff
@@ -2000,64 +2000,103 @@ export const TEXT_ENRICH_EMBED_MAX_DEPTH: number;
  */
 export declare const REGION_EVENTS: Readonly<{
   /**
-   * Triggered when the shapes or bottom/top elevation of the Region are changed.
+   * Triggered when the Region Behavior becomes active, i.e. is enabled or created without being disabled.
+   * The event is triggered only for this Region Behavior.
```

Comment:
Generally we preserve Foundry documentation as is, I'm not personally keen on editiorialising. While this is pretty minor I just wanted to point that out. There's two main places tweaks have value.
1. We switch to `@linkcode` because I think the display is better and it doesn't have any substantive change.
2. We add our own comments in `@remarks` as that's a tag Foundry doesn't use.

All in all this documentation should be:
```ts
  /**
   * Triggered when the shapes or bottom/top elevation of the Region are changed.
   *
   * @see {@linkcode foundry.documents.types.RegionRegionBoundaryEvent}
   */
```

I've noted this only here but it does apply everywhere.

Reply:
This is also pulled from the foundry api, https://foundryvtt.com/api/variables/CONST.REGION_EVENTS.html. Should I be grabbing these from somewhere else?

Reply:
Yeah, the Foundry source has greater fidelity than the Foundry documentation, I would daresay _all_ PRs should be based upon Foundry source.